docs/api/covidcast-signals/fb-survey.md

```diff
@@ -368,6 +368,17 @@ also available. These have names beginning `smoothed_w`, such as
 | `smoothed_hesitancy_reason_religious` | Estimated percentage of respondents who say they are hesitant to get vaccinated because it is against their religious beliefs, among respondents who answered "Yes, probably", "No, probably not", or "No, definitely not" when asked if they would get vaccinated if offered (item V3). This series of items was shown to respondents starting in Wave 8. <br/> **Earliest date available:** 2021-02-26 | V5abc |
 | `smoothed_hesitancy_reason_other` | Estimated percentage of respondents who say they are hesitant to get vaccinated for another reason, among respondents who answered "Yes, probably", "No, probably not", or "No, definitely not" when asked if they would get vaccinated if offered (item V3). This series of items was shown to respondents starting in Wave 8. <br/> **Earliest date available:** 2021-02-26 | V5abc |
 
+### Reasons for Believing Vaccine is Unnecessary
```

Comment:
I'd move some of the description up to a paragraph before the table. It could say this:

> Respondents who indicate that "I don't believe I need a COVID-19 vaccine" (in item V5a, V5b, V5c, or V5d) are asked a follow-up item asking why they don't believe they need the vaccine. These signals summarize the reasons selected. Respondents who do not select any reason (including "Other") are treated as missing.